graph_altimeter/scan/__init__.py

```diff
@@ -82,16 +86,19 @@ def run(config, account_id, resource_specs=None):
     not_scanned_accounts = manifest.unscanned_accounts
     logger.debug(
         "finished scan %s, \
-        scanned accounts: %s, \
-        not scanned accounts: %s",
+        scanned account: %s, \
+        not scanned account: %s",
```

Comment:
This error is a big weird when you are scanning only one account. Isn't it?

If I understand correctly, when the scan finished successfully you would get:

```
finished scan <scan_id>,        scanned account: [<account_id>],        not scanned account: []
```

and when it fails:

```
finished scan <scan_id>,        scanned account: [],        not scanned account: [<account_id>]
```

From my point of view showing both arrays does not make much sense. I would just show something like:

```
finished scan <scan_id>, scanned account <account_id>
```

or:

```
finished scan <scan_id>, could not scan account <account_id>
```

Also, the current code would add extra spaces after the commas. In cases like this it is better to do:

```python
    logger.debug(
        "finished scan %s, "
        "scanned account: %s, "
        "not scanned account: %s",
        scan_id,
        scanned_accounts,
        not_scanned_accounts,
    )
```

Reply:
Yes it makes sense, actually I think this debug line does not makes much sense now so, I think I'll remove it because, now that we are scanning one account at the same time, I think we already have enough info in the logs.